test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -16,6 +16,90 @@ public class TagHelperDescriptorFactoryTest
         private static readonly string AssemblyName =
             typeof(TagHelperDescriptorFactoryTest).GetTypeInfo().Assembly.GetName().Name;
 
+#if !DNXCORE50
```

Comment:
Suggest adding a comment about `TagHelperDesignTimeDescriptorFactory` not adding `TagHelperDesignTimeDescriptor`s when using Core. The actual restriction (around locating XML files) is a couple of layers removed from this.